### PR TITLE
[OBMIN-349]_TitleH3

### PR DESCRIPTION
**Pull request by Voloshkina**
Moved subtitle component from Storybook

### CODE REVIEW

File: src/main/resources/react/src/pages/AddGoods/selection-section/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { useEffect, useState } from 'react';
 import { useField } from 'formik';
 import { useSelector } from 'react-redux';
-import { Input, showMessage } from 'obminyashka-components';
+import { useEffect, useState } from 'react';
+import { Input, showMessage, Subtitle } from 'obminyashka-components';
```
Comment (cloudmenson) [inline src/main/resources/react/src/pages/AddGoods/selection-section/index.jsx:4]: Imports from the React should be the first, even if they are longer

File: src/main/resources/react/src/pages/AddGoods/selection-section/styles.js [hunk 2]
```diff
@@ -53,12 +49,4 @@ const Star = styled.span`
...
-export {
-  Star,
-  TitleH3,
-  Sections,
-  AddChoose,
-  InputText,
-  SectionsItem,
-  ItemDescription,
-};
+export { Star, Sections, AddChoose, InputText, SectionsItem, ItemDescription };
```
Comment (cloudmenson) [inline src/main/resources/react/src/pages/AddGoods/selection-section/styles.js:52]: Export immediately when creating component(export const)

Comment (cloudmenson) [inline src/main/resources/react/src/pages/AddGoods/selection-section/styles.js:52]: Export immediately when creating component(export const)

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

**Review by Kyslytsia — DISMISSED** [review]

**Review by ASGladchenko — APPROVED** [review]

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 3]
```diff
@@ -77,7 +72,7 @@ export const Span = styled.span`
     content: '';
     width: 1px;
     height: 11px;
-    background: #fff;
+    background: ${({ theme }) => theme.colors.white};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:75]: two callbacks
create one

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 4]
```diff
@@ -88,10 +83,10 @@ export const Span = styled.span`
 
 export const ChangeInput = styled.input`
   position: relative;
-  min-width: 262px;
+  margin: 8px 7px;
   box-sizing: border-box;
+  min-width: 262px;
   height: 32px;
-  margin: 8px 7px;
   font-family: 'Roboto', sans-serif;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:90]: Roboto is default
you can remove

File: src/main/resources/react/src/pages/AddGoods/selection-section/styles.js [hunk 5]
```diff
@@ -1,64 +1,46 @@
...
-  padding: 10px 10px 10px 16px;
-  color: #11171f;
   line-height: 26px;
   font-size: 16px;
   font-family: 'Roboto', sans-serif;
-  caret-color: ${({ theme }) => theme.colors.activeColor};
+
+  ${({ theme, p }) => css`
+    border: 1px solid
+      ${p.error ? p.theme.colors.colorError : theme.colors.borderColor};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/selection-section/styles.js:38]: check pls
Are you sure,
props p is exist?

File: src/main/resources/react/src/styledTheme/index.js [hunk 6]
```diff
@@ -19,5 +19,7 @@ export const theme = {
     activeColor: '#00C0FF',
     scrollbarBg: '#f1f1f1',
     white: '#FFFFFF',
+    colorInput: '#11171f',
+    borderColor: '#bcbcbc',
```
Comment (Wolshebnik) [inline src/main/resources/react/src/styledTheme/index.js:23]: this doesn't work

**Review by Wolshebnik — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**